config/config.json

```diff
@@ -2,21 +2,21 @@
   "development": {
     "username": "root",
     "password": null,
-    "database": "database_development",
+    "database": "todolist",
```

Comment:
他の書き方(`*_test`, `*_production`)と統一して`todolist_development` としたほうが良いでしょう？

また `todolist` だけだと `production` 環境のDBかなと勘違い知てしまう可能性もあるのでそれを防ぐ目的もあります。

Reply:
修正しました。

Reply:
config.jsonを修正した場合
DBの名前も `database_development` に変える必要があるのでしょうか？

Reply:
> DBの名前も database_development に変える必要があるのでしょうか？

現状のconfig.jsonは `todolist_development` になっていますが、`database_development` は `todolist_development` という認識でよろしいでしょうか？

利用するデータベースが変わるので、DBが存在しなければ以下のドキュメントに書かれている `db:create` を使って作成できるかと思います。

http://docs.sequelizejs.com/manual/tutorial/migrations.html

> Note: If your database doesn't exists yet, you can just call db:create command. With proper access it will create that database for you.

---

`todolist_development` DBを作成したらあとは、config.jsonの設定に合わせて `todolist_development` が使われるようになると思います。